spec/class.dd

```diff
@@ -32,18 +32,12 @@ $(GNAME ClassDeclaration):
     $(GLINK2 template, ClassTemplateDeclaration)
 
 $(GNAME BaseClassList):
-    $(D :) $(GLINK SuperClassOrInterface)
-    $(D :) $(GLINK SuperClassOrInterface) $(D ,) $(GLINK Interfaces)
+    $(D :) $(GLINK BasicType)
+    $(D :) $(GLINK BasicType) $(D ,) $(GLINK BasicTypes)
 
-$(GNAME SuperClassOrInterface):
-    $(GLINK2 type, BasicType)
-
-$(GNAME Interfaces):
-    $(GLINK Interface)
-    $(GLINK Interface) $(D ,) $(GSELF Interfaces)
-
-$(GNAME Interface):
-    $(GLINK2 type, BasicType)
```

Comment:
SuperClassOrInterface and Interfaces are still referenced by AnonBaseClassList and BaseInterfaceList.